execution/executor/src/chunk_executor/mod.rs

```diff
@@ -597,13 +598,21 @@ impl<V: VMExecutor> ChunkExecutorInner<V> {
             .map(|t| t.into())
             .collect::<Vec<SignatureVerifiedTransaction>>();
 
+        let global_cache_manager = GlobalCacheManager::new_with_default_config();
+        global_cache_manager.mark_block_execution_start(&state_view, None)?;
```

Comment:
If I understand the code correctly, you are creating new GlobalCacheManager here on every block? so there is no caching happening with this PR?

as I pointed here, [my code](https://github.com/aptos-labs/aptos-core/pull/15137/files#diff-37328268a2d6d43154879be178a20390088e344848f2babf19a38ea64c0d854fR602) changes below to use an instance of V (instead of static methods), and so GlobalCacheManager can be inside of V. 
But you still need to have some way to create V (with GlobalCacheManager inside) once (i.e. once per epoch), and reuse it across blocks. 


Also, note that V is generic here. Only when V is AptosVMBlockExecutor does having GlobalCacheManager manager makes sense. and from that perspective as well - it is better to have mark_block_execution_start / mark_block_execution_end be called from inside AptosVMBlockExecutor, isntead of here

Reply:
actually this is just verify